eox_core/pipeline.py

```diff
@@ -80,13 +91,20 @@ def force_user_post_save_callback(auth_entry, is_new, user=None, *args, **kwargs
     The first one was selected because the second executed multiple unnecessary process and third one
     needs to implement a new backend.
     """
+    is_new = kwargs.get('is_new')
+
     if auth_entry == 'register' and user and is_new:
         user._changed_fields = {'is_active': user.is_active}  # pylint: disable=protected-access
         post_save.send_robust(
             sender=user.__class__,
             instance=user,
             created=False
         )
+        logging_pipeline_step(
```

Comment:
@morenol Just wanted to let you know that I added a log for this step

Reply:
It looks fine! @magajh